scripts/maasutils.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import click
+import pyrax
+import requests
+
+
+@click.option("--username", required=True)
+@click.option("--api-key", required=True)
+@click.group()
+def cli(api_key, username):
+    pyrax.set_setting("identity_type", "rackspace")
+    pyrax.set_credentials(
+        username,
+        api_key
+    )
```

Comment:
The monitoring api doesn't split by regions as the others do. You can view and modify entities that link to instances from any region, regardless of the region specified on connection. As this pyrax connection is only used for maas, specifying the region is superfluous and potentially misleading.